karma.conf.js

```diff
@@ -8,7 +11,10 @@ function depMain(name) {
 }
 
 //  dependencies :: Array String
-var dependencies = Object.keys(require('./package.json').dependencies);
+var dependencies = ['sanctuary-type-classes', 'sanctuary-def'];
+
+eq(dependencies.slice().sort(),
+   Object.keys(require('./package.json').dependencies).sort());
```

Comment:
Yea this is probably the better way now that I think about it, just having a test check if things are out of sync.

👍